handlers/update_resource_test.go

```diff
@@ -0,0 +1,79 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/appleboy/gofight"
+	"github.com/stretchr/testify/assert"
+	"github.com/sul-dlss-labs/taco/persistence"
+)
+
+func TestUpdateResourceHappyPath(t *testing.T) {
+	r := gofight.New()
+	repo := mockRepo(new(persistence.Resource))
+
+	newId := ""
+
+	r.POST("/v1/resource").
```

Comment:
Rather than making a POST request here, since this is a unit test, can you call `rt.Repository().SaveItem()` thus removing the dependency on `handlers/deposit_resource.go`?

Reply:
This looks like it would require a change to SaveItem() to return the ID (currently returns nil). Are we OK with that change and what impact might it have?

Reply:
SaveItem uses whatever ID you pass into it. So you can just pass in "9999" or something.

Reply:
Actually, this is a unit test, so you should just use a sub implementation of repository.  There should be no need to create the object before hand.